docs/withdraw_manually.md

```diff
@@ -0,0 +1,108 @@
+# Withdraw Funds Manually
+
+This guide explains how to withdraw funds stored in:
+
+- **Agent Safe**
+- **Agent EOA**
+- **Master Safe** (includes agent bonds and security deposits)
+- **Master EOA**
+
+## Prerequisites
+
+- You have [Metamask](https://metamask.io/) installed in your browser and know how to execute transactions.
+- You understand Safe{Wallet}s and have some familiarity with the [Safe webapp](https://app.safe.global/).
+
+> [!IMPORTANT]
+> This guide does **not** cover removing external investments (e.g., liquidity pools) made by the agent. Ensure that the agent has withdrawn any such funds to the **Agent Safe** before proceeding.
+
+## Step 1: Withdraw from Agent EOA and Agent Safe
+
+To withdraw funds from the Agent EOA and Agent Safe you have to access contents stored in the `.operate` folder:
+
+- **Mac**: `/Users/YOUR_USERNAME/.operate`
+- **Windows**: `C:\Users\YOUR_USERNAME\.operate`
+- **Linux**: `/users/YOUR_USERNAME/`
+
+1. **Stop Pearl.**
+
+2. **Locate Agent Data.** Open the file `.operate/sc-xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/config.json` in a text editor. If running multiple agents, ensure to select the correct `sc-xxxxxx...` folder. Search for:
+   1. `name`: Identifies the agent type (e.g., Trader, Optimus, ...).
+   2. `keys`: Contains the **Agent EOA address** and its **private key**.
+   3. `multisig`: Contains the **Agent Safe address**.
+   4. `token`: Contains the **Ownership NFT ID**.
+
+3. **Import the Agent EOA into Metamask.**
+   1. Copy the **Agent EOA private key**.
+   2. Open Metamask.
+   3. Go to *Add account or hardware wallet* &#8594; *Import account*.
+   4. Select *Private Key*, paste the key, and press *Import*.
+   5. Ensure the **Agent EOA address** is selected and connected to the correct chain.
+
+4. **Withdraw from Agent Safe**
+   1. Open the Safe webapp: https://app.safe.global/home?safe=gno:AGENT_SAFE_ADDRESS (replace `AGENT_SAFE_ADDRESS`).
```

Comment:
maybe
```suggestion
   1. Open the Safe webapp: https://app.safe.global/home?safe=gno:AGENT_SAFE_ADDRESS (replace `AGENT_SAFE_ADDRESS` with `multisig` from Step 1.2).
```

Reply:
also should mention chain, gno is for gnosis
and that it's not supported on mode

Reply:
Actioned.